vendordeps/SdsSwerveLib.json

```diff
@@ -0,0 +1,19 @@
+{
+  "fileName": "SdsSwerveLib.json",
+  "name": "swerve-lib",
+  "version": "aac04effce119d4ad6867052ce2c003ba10917f4",
+  "uuid": "9619F7EA-7F96-4236-9D94-02338DFED592",
+  "mavenUrls": [
+    "https://jitpack.io"
+  ],
+  "jsonUrl": "https://raw.githubusercontent.com/SwerveDriveSpecialties/swerve-lib/master/SdsSwerveLib.json",
+  "javaDependencies": [
+    {
+      "groupId": "com.github.SwerveDriveSpecialties",
+      "artifactId": "swerve-lib",
+      "version": "aac04effce119d4ad6867052ce2c003ba10917f4"
+    }
```

Comment:
This is an older version, and https://github.com/SwerveDriveSpecialties/swerve-lib/commits/develop shows a subsequent fix for inverted sensors that's in the 1.0.0 version.  Updating this file to the current one from https://raw.githubusercontent.com/SwerveDriveSpecialties/swerve-lib/master/SdsSwerveLib.json should address this.